src/Core/src/Handlers/Label/LabelHandler.Windows.cs

```diff
@@ -1,23 +1,48 @@
 using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class LabelHandler : AbstractViewHandler<ILabel, TextBlock>
+	public partial class LabelHandler : AbstractViewHandler<ILabel, FrameworkElement>
 	{
-		protected override TextBlock CreateNativeView() => new TextBlock();
+		protected TextBlock? TextBlock { get; set; }
 
-		public static void MapText(IViewHandler handler, ILabel label) =>
-			(handler as LabelHandler)?.TypedNativeView?.UpdateText(label);
+		protected override FrameworkElement CreateNativeView()
+		{
+			TextBlock = new TextBlock();
+			return new Border { Child = TextBlock };
```

Comment:
Until a better way with the Container feature, this works. We should change this and use a container since this is a bit unexpected. However, not very different from say an Android search bar where the entry is a child of the "layout" control.